packages/stark-ui/src/modules/input-mask-directives/directives/email-mask.directive.spec.ts

```diff
@@ -16,8 +16,8 @@ describe("EmailMaskDirective", () => {
 		template: getTemplate("[starkEmailMask]='emailMaskConfig'")
 	})
 	class TestComponent {
-		public emailMaskConfig: boolean;
-		public ngModelValue: string = "";
+		public emailMaskConfig?: boolean;
```

Comment:
Now with this change you can remove some useless casts like `hostComponent.emailMaskConfig = <any>undefined;`